FrontEnd/src/Components/CardData/Midjourney.js

```diff
@@ -6,6 +6,28 @@ const cardData = [
     imageAlt: "Loading Image...",
     title: "Cute Doodle Art Illustrations",
     tag: "⛵ Midjourney",
+    description: `This prompt generates images of various objects in a doodle style.
+        Each image is unique and creatively designed, featuring simple,
+        hand-drawn lines and shapes that create a playful, whimsical feel.
+        Whether you&apos;re looking for inspiration for your next art project
+        or just want to have fun exploring your imagination, this prompt is
+        perfect for creating a diverse range of doodle-style images. Perfect
+        for profile pics, storybooks, art prints, gifts, concept art and to
+        have fun. They have a friendly hand-drawn look and are often found on
+        different type of digital design backgrounds. Stop paying for stock
+        images for this purpose and generate your own with the type of subject
+        that you want.`,
+    model: "Midjourney v4",
+    originalprompt:
```

Comment:
Please keep the textcase same for all the properties(keys), currently I can see a mix of camelCase and lowercase

Reply:
Got it, will make changes!